draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -260,74 +259,21 @@ TLS:  Transport Layer Security
-   buffering allocated for DTLS processing as well as the DTLS/SCTP
-   API to the application. To ensure that the sender have some
-   understanding of the maximum receiver size a TLS extension
-   "dtls_over_sctp_maximum_message_size" {{TLS-Extension}} is used to
-   signal the endpoints receiver capability when it comes to user
-   message size.
-
-   All implementors of this specification MUST support user messages
-   of at least 16383 bytes. Where 16383 bytes is the supported message
-   size in RFC 6083. By requiring this message size in this document,
-   we ensure compatibility with existing usage of RFC 6083, not
-   requiring the upper layer protocol to implement additional features
-   or requirements.
-
-   Message size limits due to memory limitation or other criteria
-   and the related interaction towards the SCTP users are specific to the
-   implementations.
-
-   Due to SCTP's capability to transmit concurrent user messages the
-   total memory consumption in the receiver is not bounded. In cases
-   where one or more user messages are affected by packet loss, the
-   DATA chunks may require more data in the receiver's buffer.
-
-   The necessary buffering space for a single user message of
-   dtls_over_sctp_maximum_message_size (MMS) is dependent on the
-   implementation.
-
-   When no partial data delivery is supported, the message size is
-   limited by the a_rwnd as this is the largest protected user message
-   that can be received and then processed by DTLS and where the plain
-   text user message is expected to be no more than the signalled MMS.
-
-   With partial processing it is possible to have a receiver
-   implementation that is bound to use no more buffer space than MMS
-   (for the plaintext) plus one maximum size DTLS record. The later
-   assumes that one can realign the start of the buffer after each
-   DTLS record has been consumed. A more realistic implementation is
-   two maximum DTLS record sizes.
-
-   If an implementation supports partial delivery in both the SCTP API
-   and the ULP API, and also partial processing in the DTLS/SCTP
-   implementation, then the buffering space in the DTLS/SCTP layer
-   ought to be no more than two DTLS records. In which case the MMS to
-   set is dependent on the ULP and the endpoints capabilities.
-
-   In cases where not all DTLS records of a protected user messages is
-   delivered and partial delivery is not supported, nothing will be
-   delivered to the ULP. If partial deliver is supported the ULP will
-   be informed that the received message is not complete.
-
+   securely. However, by requiring the use of the partial delivery of
+   user messages from SCTP {{RFC6458}} the required buffering prior to
+   DTLS processing can be limited to a single DTLS record per user
+   message (unordered delivery streams) and used streams (ordered
+   delivery streams). This enables the DTLS/SCTP implementation to provide the
```

Comment:
Ordered or unordered is a property of a user message, not of a stream. So I would suggest to write something like:
```
However, by requiring the use of the partial delivery of user messages
from SCTP and assuming that no two messages received on the same
stream are interleaved (as it is the case when using the API defined
in {{RFC6458}}) the required buffering prior to DTLS processing can
be limited to a single DTLS record per used incoming stream.
```